code/src/components/Message.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/react-in-jsx-scope */
+import { useEffect, useState } from 'react';
+import './Message.css';
+
+const dayjs = require('dayjs');
+const relativeTime = require('dayjs/plugin/relativeTime');
+
+dayjs.extend(relativeTime);
+
+export default function Message(props) {
+  const [clickedId, setClickedId] = useState(null);
+  const [likeValue, setLikedValue] = useState(props.likeValue);
+  const [isLiked, setIsLiked] = useState();
+
+  useEffect(() => {
+    if (isLiked) {
+      // POST request using fetch inside useEffect React   hook
+      const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' }
+      };
+      fetch(
+        `https://happy-thoughts-technigo.herokuapp.com/thoughts/${clickedId}/like`,
+        requestOptions
+      )
+        .then((response) => response.json())
```

Comment:
I noticed that in different fetch request you called `reponse` argument `raw` - let's be consequent about our naming, change either this or that argument name so they match